.woodpecker/test.yml

```diff
@@ -64,9 +65,12 @@ steps:
   lint-editorconfig:
     image: mstruebing/editorconfig-checker:2.7.2
     group: test
+    when:
+      - event: [pull_request, tag, deployment]
```

Comment:
I think we can remove `deployment` from this file completely as we don't use it.